index.md

```diff
@@ -94,12 +94,13 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 5. A description MUST immediately follow the type/scope prefix.
 The description is a short description of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
 6. A longer commit body MAY be provided after the short description, providing additional contextual information about the code changes. The body MUST begin one blank line after the description.
-7. A footer MAY be provided one blank line after the body (or after the description if body is missing).
+7. The description and commit body MUST be written in [imperative](https://en.wikipedia.org/wiki/Imperative_mood) present tense.
```

Comment:
Do you think that it _MUST be writter_?

Reply:
Not sure what you mean.

Are you asking whether the imperative form should be a requirement of the convention?

Reply:
Do you think that we should use _MUST_? I'd prefer _SHOULD_?
I know Several people that are "against" this way of thinking, so I think _SHOULD_ it's better.

Reply:
@damianopetrungaro you are right. I did [say that I would _recommend_ the imperative form](https://github.com/conventional-commits/conventionalcommits.org/pull/65#issuecomment-412357437).

I have updated the pull request.

Reply:
Thank you for the effort @hbetts !